packages/vite/src/node/plugins/css.ts

```diff
@@ -560,7 +560,7 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           .replace(/\./g, '\\.')
         const emptyChunkRE = new RegExp(
           opts.format === 'es' || opts.format === 'system'
-            ? `\\bimport\\s*"[^"]*(?:${emptyChunkFiles})";\n?`
+            ? `\\bimport\\s*["'][^"]*(?:${emptyChunkFiles})["'];\n?`
             : `\\brequire\\(\\s*"[^"]*(?:${emptyChunkFiles})"\\);\n?`,
```

Comment:
```suggestion
            ? `\\bimport\\s*["'][^"']*(?:${emptyChunkFiles})["'];\n?`
            : `\\brequire\\(\\s*["'][^"']*(?:${emptyChunkFiles})["']\\);\n?`,
```
I think this is better because

- If `[^"]` is used instead of `[^"']`, it matches `import 'chunkName'; console.log(); 'chunkName';`.
- This would also happen with `require`.

Reply:
@matthewp I think you aren't letting maintainers do changes in the PR, no? So I can't commit this myself.

Reply:
Anyways, I added this suggestion.

Reply:
There is a [check to allow maintainers to make changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) that may not be enabled for this PR